### PR TITLE
Improve a11y

### PR DESCRIPTION
**Pull request by nvh95**
This PR is to improve accessibility when visiting https://2022.jsconf.kr/en. Details on each one will be commented directly in the File Changes

The improvement by Google Devtools LightHouse: **84 => 100**

**Before**
accessibility score of 84 before the update

**After**
accessibility score of 100 after the update

### CODE REVIEW

File: src/pages/components/Footer.tsx [hunk 1]
```diff
@@ -95,7 +95,7 @@ export function Footer() {
 
         {/* CopyRight */}
         <div className="flex gap-4 place-self-center sm:order-2">
-          <span className="text-xs text-[#404040]">
+          <span className="text-xs text-[#919191]">
```
Comment (nvh95) [inline src/pages/components/Footer.tsx:98]: Increase the contrast

File: src/pages/components/Navbar.tsx [hunk 2]
```diff
@@ -75,8 +75,9 @@ export function Navbar() {
         </ul>
 
         {/* Mobile */}
-        <label className="btn swap btn-circle swap-rotate border-none bg-transparent hover:bg-transparent lg:hidden">
+        <label className="swap-rotate btn swap btn-circle border-none bg-transparent hover:bg-transparent lg:hidden">
```
Comment (nvh95) [inline src/pages/components/Navbar.tsx:78]: reorder classes by `prettier-plugin-tailwindcss` (https://github.com/tailwindlabs/prettier-plugin-tailwindcss)

File: src/pages/components/Navbar.tsx [hunk 3]
```diff
@@ -75,8 +75,9 @@ export function Navbar() {
...
           <input
+            aria-label="Open Menu"
```
Comment (nvh95) [inline src/pages/components/Navbar.tsx:80]: Add an aria-label on the hamburger icon when opening on the small screen.

File: src/pages/components/Navbar.tsx [hunk 4]
```diff
@@ -106,7 +107,7 @@ export function Navbar() {
       <div className="">
         <ul
           className={`menu menu-vertical fixed top-0 left-0 z-30 flex h-screen h-[100svh] max-h-screen w-full justify-start justify-items-start bg-black pt-[88px] pb-12 text-lg text-white transition-opacity duration-300 lg:hidden${
-            isOpen ? ' opacity-100' : ' pointer-events-none opacity-0'
+            isOpen ? ' opacity-100' : 'pointer-events-none hidden opacity-0'
```
Comment (nvh95) [inline src/pages/components/Navbar.tsx:110]: Avoid items in the overlay being read by screen reader if they are not rendered, by adding `hidden` class

File: src/pages/components/Performs.tsx [hunk 5]
```diff
@@ -26,7 +26,10 @@ export const Performs = () => {
                   src={`/${perform}.profile.png`}
                   alt={t(`${perform}.name`)}
                 />
-                <span className="text-center text-xs text-slate-400 sm:text-sm">
+                <span
+                  aria-hidden
+                  className="text-center text-xs text-slate-400 sm:text-sm"
+                >
```
Comment (nvh95) [inline src/pages/components/Performs.tsx:32]: Avoid names of MCs are read twice (Already read by `Image`'s alt text)

File: src/pages/components/Speakers.tsx [hunk 6]
```diff
@@ -51,6 +51,7 @@ export const Speakers = () => {
                     href={t(`${speaker}.link.slide`)}
                     target="_blank"
                     rel="noopener noreferrer"
+                    aria-label="Slides"
```
Comment (nvh95) [inline src/pages/components/Speakers.tsx:54]: Add label for new icon `Slides`

File: src/pages/components/Speakers.tsx [hunk 7]
```diff
@@ -90,7 +91,9 @@ export const Speakers = () => {
                   </svg>
                 </div>
                 <h3 className="text-[13px] font-semibold leading-4 sm:text-base">
-                  {t(`${speaker}.title`)}
+                  <Link href={`/speakers/${speaker}`}>
+                    <a>{t(`${speaker}.title`)}</a>
+                  </Link>
```
Comment (nvh95) [inline src/pages/components/Speakers.tsx:96]: Upgrading in UX if user just `tab` though the page. Screen reader will read name of speaker, then name of the presentation

**Review by nvh95 — COMMENTED** [review]
Hi @ysm-dev @mojosoeun 

I just added some minor changes to improve the accessibility of https://2022.jsconf.kr/en. Please let me know if they are good. Thanks.

**Comment by mojosoeun** [comment]
@nvh95 Hi, Hung Thank you so much improving our website's a11y

**Pull request merged**